core/src/main/java/com/linecorp/armeria/server/ServiceRequestContext.java

```diff
@@ -258,6 +260,28 @@ default String pathParam(String name) {
         return pathParams().get(name);
     }
 
+    /**
+     * Returns the query parameters mapped by the {@link Route} associated with the {@link Service}
+     * that is handling the current {@link Request}.
```

Comment:
Unlike `pathParams`, query parameters are not heavily used by `Route`. Although it is possible.
Could we explain where/how this value came from? 
```
Returns the decoded {@linkplain QueryParams query part} of the current {@link Request} URI,
as defined in <a href="https://datatracker.ietf.org/doc/rfc3986/">RFC3986</a>.
```